astra_mini.launch.py

```diff
@@ -0,0 +1,85 @@
+from launch import LaunchDescription
```

Comment:
Here copyright should be mentioned.

Reply:
There is no licence: https://github.com/orbbec/ros2_astra_camera/tree/master